RELEASE_NOTES.md

```diff
@@ -9,9 +9,13 @@ For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
 ## 12.4.0 (Unreleased)
 
+### `oc-id`
+    * Pulled in a new version of Rails for `oc-id` due to critical vulnerabilities found in Rails: http://weblog.rubyonrails.org/2016/1/25/Rails-5-0-0-beta1-1-4-2-5-1-4-1-14-1-3-2-22-1-and-rails-html-sanitizer-1-0-3-have-been-released/
+
 ### Chef Server
   * There is now a Universe endpoint, which provides the same output
     as the Supermarket or berkshelf-api universe endpoints.
+  * Implemented Server Admins. Server Admins is a list of users that that full permissions access on the users endpoint. You can add and remove users from the Server Admins list. This allows non-superusers to perform creation, deletion, editing, and listing on all users (except the superuser) in a Chef Server, granting great flexibility around user management.
```

Comment:
Also they both will need to learn how to download the cookbooks (at least for the policyfile case) since other HTTP services that they use serve tarballs.